main.go

```diff
@@ -24,7 +24,7 @@ import (
 	"flag"
 	"fmt"
 
-	"github.com/kubermatic/k8sniff/metrics"
+	"github.com/platform9/k8sniff/metrics"
```

Comment:
Let's keep the code as close to upstream as possible. Just make sure the repo is checked out at $GOPATH/src/kubermatic